packages/cascara/src/ui/Columns/Columns.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import pt from 'prop-types';
+import { Role } from 'reakit';
+import classNames from 'classnames/bind';
+
+import Boundaries from '../../system-components/Boundaries';
+import styles from './Columns.module.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /** The section content */
+  children: pt.oneOfType([pt.arrayOf(pt.node), pt.node]),
+  /** Columns can have css class name */
+  className: pt.string,
+  /** The number of columns to allocate */
+  count: pt.number,
+};
+
+const Columns = ({ count = 2, children, className, ...rest }) => (
+  <Boundaries>
+    <Role
```

Comment:
@lordpixel because we are using the Role component, I think we should be exposing the `as` prop (with a default as well) to developers. The default `as` in this scenario should be `'div'`
```suggestion
    <Role
      as={as}
```

Reply:
@brianespinosa I did not commit the suggestion because we also needed to add props definition for `as`, so I went and did it locally. 👍